frontend/src/components/auth/auth.tsx

```diff
@@ -48,6 +48,9 @@ const Auth: FC = () => {
           />
         );
       }
+      case AppRoute.STUDIO: {
```

Comment:
It shouldn't be in the authorization component

Reply:
Done!